README.md

```diff
@@ -18,6 +18,15 @@ This is because `gcloud` and `az` are very large packages and we want to keep th
 - `spacelift-io/runner-terraform:gcp-latest` -> with `gcloud` CLI
 - `spacelift-io/runner-terraform:azure-latest` -> with `az` CLI
 
+### FIPS
+
+All the tags have a `-fips` variant that has been installed with the latest FIPS compliant OpenSSL (3.1.2 at the time of writing).
+You are still responsible to run these images on a FIPS hardened host ensuring youre only communicating with FIPS enabled endpoints.
+
+- `spacelift-io/runner-terraform:latest-fips` -> with `aws` CLI & FIPS OpenSSL
+- `spacelift-io/runner-terraform:gcp-latest-fips` -> with `gcloud` CLI & FIPS OpenSSL
+- `spacelift-io/runner-terraform:azure-latest-fips` -> with `az` CLI & FIPS OpenSSL
```

Comment:
Just one nit, feel free to ignore. Would it make sense to name these `fips-latest`, `gcp-fips-latest` and `azure-fips-latest`? Keeping the `latest` adjective to the env of the tag.

Reply:
I did this change and then I reverted it, I have 2 reasons for this IMO. 

1. Most image providers I've come across will use `-fips` at the end of their tags to signify a fips image.
2. I made the change, and it makes the code a little confusing. Because some places it needs to be at the end (like in the trivy scans, or in the bake target) and then others it would need to be in the middle. This way, everything is just at the end.